monkey/monkey_island/cc/resources/auth/auth.py

```diff
@@ -42,6 +45,7 @@ def post(self):
 
         if _credentials_match_registered_user(username, password):
             access_token = _create_access_token(username)
+            _check_attack_mitigations_in_mongo()
```

Comment:
This checks for attack mitigations during login? Why? Shouldn't it happen during registration?